op-node/rollup/derive/l1_block_info.go

```diff
@@ -273,8 +273,20 @@ func L1InfoDeposit(rollupCfg *rollup.Config, sysCfg eth.SystemConfig, seqNumber
 	var err error
 	if isEcotoneButNotFirstBlock(rollupCfg, l2BlockTime) {
 		l1BlockInfo.BlobBaseFee = block.BlobBaseFee()
-		l1BlockInfo.BlobBaseFeeScalar = sysCfg.BlobBaseFeeScalar
-		l1BlockInfo.BaseFeeScalar = sysCfg.BaseFeeScalar
+		if l1BlockInfo.BlobBaseFee == nil {
+			// The L2 spec states to use the MIN_BLOB_GASPRICE from EIP-4844 if not yet active on L1.
+			l1BlockInfo.BlobBaseFee = big.NewInt(1)
```

Comment:
nit: could use a constant like `common.Big1` from geth to save an alloc

Reply:
We've tried to move away from `common.Big1` because they can accidentally be mutated.